chirp/drivers/uvk5.py

```diff
@@ -1013,6 +1016,11 @@ def get_memory(self, number2):
         else:
             mem.empty = False
 
+        # FIXME: This should set duplex='off' for channels that are outside
```

Comment:
@egzumer Can you tell me what a reasonable bound is for when we should allow the radio to transmit? I assume it can't actually transmit on 27MHz, right?

Reply:
It depends on radio settings (modulation, unlock settings, etc). If the settings are right you can transmit on any freq you can input.

Reply:
Ah, okay, well, I'll just leave it as-is then.